clang/include/clang/CIR/CIRGenerator.h

```diff
@@ -53,6 +53,7 @@ class CIRGenerator : public clang::ASTConsumer {
   ~CIRGenerator() override;
   void Initialize(clang::ASTContext &astCtx) override;
   bool HandleTopLevelDecl(clang::DeclGroupRef group) override;
+  mlir::ModuleOp getModule();
```

Comment:
Is there a reason this isn't a `const` member function?

Reply:
https://mlir.llvm.org/docs/Rationale/UsageOfConst/

Types that implement MLIR data structures are not `const` correct.  That attitude will probably bleed over into ClangIR types that aren't part of the MLIR dialect.  But in this case I think this particular function can be `const`.

Reply:
Fixed.  I changed `CIRGenerator::getModule()` to be a `const` member function.